src/exceptions.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+use rquickjs::{
+    atom::PredefinedAtom,
+    function::{Constructor, Opt},
+    Class, Ctx, Object, Result,
+};
+
+#[rquickjs::class]
+#[derive(rquickjs::class::Trace)]
+struct DOMException {
+    message: String,
+    name: String,
+    stack: String,
+}
+
+#[rquickjs::methods]
+impl DOMException {
```

Comment:
Only toString left and we're good to go! Nice job!

Reply:
In other words, toString() is unnecessary!

Reply:
It's not in spec because of `Error` inheritance. Since we're not really inheriting just adding to prototype chain we must manually implement it as well

Reply:
What do you mean by "manually implement it"?
I believe we are probably missing a test case, since all of the test cases we currently have in place are passing.
What test cases should we create to detect the lack of manual implementation?

Reply:
I mean add it to the `impl` and create a test case for it :)